int8 casualty_id
int8[] trauma_head
int8[] trauma_torso
int8[] trauma_lower_ext
int8[] trauma_upper_ext
int8[] alertness_ocular
int8[] severe_hemorrhage
int8[] motion
